examples/plot_bcic_iv_2a_moabb_trial.py

```diff
@@ -158,7 +158,9 @@
     torch.backends.cudnn.benchmark = True
 # Set random seed to be able to roughly reproduce results
 # Note that with cudnn benchmark set to True, GPU indeterminism
-# may still make results substantially different between runs
+# may still make results substantially different between runs.
+# To obtain more consistent results you can set `cudnn_benchmark=False` in `set_random_seeds`
```

Comment:
```suggestion
# To obtain more consistent results at the cost of increased computation time,
# you can set `cudnn_benchmark=False` in `set_random_seeds`
```

Reply:
Maybe make it now fully complete @sliwy  :D ?

Reply:
Done :D